usaspending_api/api_contracts/contracts/v2/download/disaster.md

```diff
@@ -9,14 +9,10 @@ Creates a new download job for the requested account and award. Returns a link t
 
 + Request (application/json)
     + Attributes (object)
-        + `filters` (required, Filters, fixed-type)
+        + `filters` (optional, Filters, fixed-type)
```

Comment:
Should `def_codes` be listed in the filters or is the example below incorrect?

Reply:
Line 13: `Currently ignored, download will only be generated for COVID-19 spending data`

Currently filters are completely ignored

Reply:
egg... all... over.... **my**... face...

In my defense, though, I can't read.